ckanext/xloader/action.py

```diff
@@ -166,15 +173,20 @@ def xloader_submit(context, data_dict):
         job = enqueue_job(
             jobs.xloader_data_into_datastore, [data], queue=custom_queue,
             title="xloader_submit: package: {} resource: {}".format(resource_dict.get('package_id'), res_id),
-            rq_kwargs=dict(timeout=timeout)
+            rq_kwargs=dict(timeout=timeout, at_front=sync)
```

Comment:
what version had at_front added to the api?

Reply:
@duttonw the `rq_kwargs` are to pass to the `rq` lib. So not sure when rq added at_front

Reply:
@duttonw https://github.com/rq/rq/blob/master/CHANGES.md#050 <- okay so it was added in 2015. In version `0.5.0` and CKAN 2.8 was using `rq==0.6.0` so we should be all good with this usage.